AdminServer/appscale/admin/push_worker_manager.py

```diff
@@ -139,8 +139,9 @@ def _write_worker_configuration(self, queue_config):
       rates = {'default': '5/s'}
     else:
       queues = json.loads(queue_config)['queue']
-      rates = {queue: queues[queue]['rate'] for queue in queues
-               if 'mode' not in queue or queue['mode'] == 'push'}
+      rates = {
+        queue_name: queue['rate'] for queue_name, queue in queues.items()
+        if 'mode' not in queue or queue['mode'] == 'push'}
```

Comment:
But this change doesn't modify content of resulting dict - old version and new are equal.
Does PR name correspond to it's content?